examples/README.md

```diff
@@ -2,6 +2,8 @@
 
 This directory contains example projects built with the HONC stack, showcasing different use cases, patterns, and integrations.
 
+> 📚 For more examples and templates, check out the [Awesome Honc](https://github.com/fiberplane/awesome-honc) repository.
```

Comment:
would be good to add this to the main repo README too - it'd get more eyes on it 👀